wesnoth-trow.po

```diff
@@ -5561,46 +5426,33 @@ msgstr ""
 
 #. [message]: speaker=Lord Aryad
 #: data/campaigns/The_Rise_Of_Wesnoth/scenarios/18_A_Spy_in_the_Woods.cfg:128
-#, fuzzy
-#| msgid ""
-#| "We may have decided wrongly. If the humans drive the orcs off we should "
-#| "honor our pledge. If more trouble follows them here we should let them "
-#| "fight it out, then we should ‘deal’ with the survivors, and make a "
-#| "compromise with the Dwarves."
 msgid ""
 "We may have decided wrongly. If the humans drive the orcs off we should "
 "honor our pledge. If more trouble follows them here we should let them fight "
 "it out, then we should ‘deal’ with the survivors, and make a compromise with "
 "the dwarves."
 msgstr ""
 "Možná jsme rozhodli špatně. Pokud ti lidé skřety vytlačí, měli bychom držet "
-"svůj slib. Budou-li následovat další potíže, měli bychom je nechat, ať si je "
-"vyřeší, a pak „jednat“ s přeživšími a uzavřít smír s trpaslíky."
+"svůj slib. Budou-li je sem následovat další potíže, měli bychom je nechat si "
+"je vyřešit, a pak „jednat“ s přeživšími a uzavřít smír s trpaslíky."
```

Comment:
Až sem mi připadaly všechny opravy jako velké zlepšení. Díky za každou z nich, včetně těch, co se týkaly mých neumělých překladů.

Tady mi ale to přidání třetího **je** a zrušení věty vložené, která to poslední **je** alespoň trochu oddělila, jako zlepšení nepřipadá.

Pokud to tak chcete nechat, je třeba zrušit čárku.
```suggestion
"svůj slib. Budou-li je sem následovat další potíže, měli bychom je nechat si "
"je vyřešit a pak „jednat“ s přeživšími a uzavřít smír s trpaslíky."
```

Reply:
K těmhle dvěma větám (vlastně k jedné, dvakrát citované):
Je možné, že jsem se moc soustředil na to, aby byl stejný textový řetězec shodně přeložen a zazdil jsem stylistickou úroveň.  Klidně to lze předělat, jen bych chtěl, aby to bylo v obou případech stejně.

Reply:
Také si dovolím podotknout, že ta věta: „If more trouble follows them here we should let them fight it out, then we should ‘deal’ with the survivors, and make a compromise with the Dwarves“ není správně přeložená tím původním: „Budou-li následovat další potíže, měli bychom je nechat, ať si je vyřeší, a pak „jednat“ s přeživšími a uzavřít smír s trpaslíky“.
Očividně se tam mluví o potížích, které mohou JE (ty lidi) SEM (do této země) následovat. To je geneze toho třetího JE.
Přemýšlím o nějaké třetí variantě...

Reply:
Třeba:
Budou-li je sem následovat další potíže, měli bychom je nechat, _aby jim čelili sami_ a pak se „vypořádat“ s přeživšími a uzavřít smír s trpaslíky.
Prostě nahradit to _je vyřešit_ jiným opisem.

Nebo:
_Přitáhnou-li sem ale další potíže_, měli bychom je nechat...
Nahradit _budou-li je_ jiným opisem.

Reply:
Vy jste se rozjel. Cokoli s tím uděláte, bude dobře. Už při prvním čtení jsem pochopil, proč jste tam to první **je** přidal. Asi mi vadí to **je nechat si je**, kde druhé **je** znamená lidi a třetí, blízké **je** znamená potíže. Ale nejvíc mi vadila ta čárka, která tam zůstala.

Takže to nechte, jak to je, a zrušte čárku, nebo, pokud je vám líto opisných návrhů, použijte který se vám nejvíc líbí. Abyste věděl, co nechcete, připojuji svůj návrh:

Pokud za nimi přitáhnou další potíže, měli bychom je nechat, ať si to vybojují sami, pak bychom se měli „vypořádat“ s přeživšími a dohodnout se s trpaslíky.

Reply:
Náhodou ten tvůj poslední návrh se mi líbí nejvíc. Je to přesné a uhlazené.

Reply:
Dík, teda. Tak já to požeru, opravím u sebe (konec je moc přepálený, tak ho ještě o pár slov odlehčím) a vrznu zpátky. Nestyďte se do toho ještě sáhnout, když se vám to nebude líbit.